lib/CUDD/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 set(SOURCES
        cuddAddNeg.c
        cuddBddCorr.c
-       cuddDecomp.c
+       cuddDecomp.cpp
```

Comment:
CUDD-2.5.0 is a pretty old lib. I believe almost all the CUDD files were written before 2010 (std11). For example, [dddmpStoreAdd.c](https://github.com/unsw-corg/SVF/blob/master/lib/CUDD/dddmpStoreAdd.c) you changed to `cpp` file was written in 2004. There is no std11 feature there:)

In order to make all the CUDD files consistent, could you please leave CUDD file extensions unchanged?

Thanks